Modules/IO/MINC/src/itkMINCImageIO.cxx

```diff
@@ -497,13 +505,29 @@ MINCImageIO::ReadImageInformation()
       sep[i - 1] = _sep;
 
       this->SetDimensions(i - 1, static_cast<unsigned int>(_sz));
-      this->SetDirection(i - 1, _dir);
       this->SetSpacing(i - 1, _sep);
 
       ++usable_dimensions;
     }
   }
 
+  // Transform MINC PositiveCoordinateOrientation RAS coordinates to
+  // internal ITK PositiveCoordinateOrientation LPS Coordinates
+  dir_cos = RAS_tofrom_LPS * dir_cos;
+
+  // Transform origin coordinates
+  o_origin = dir_cos * origin;
```

Comment:
Same comment as above concerning `o_origin` and any other variable that this patch set is adding.